src/main/java/mate/academy/spring/service/impl/UserServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.spring.service.impl;
+
+import java.util.List;
+import mate.academy.spring.dao.UserDao;
+import mate.academy.spring.model.User;
+import mate.academy.spring.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class UserServiceImpl implements UserService {
+    @Autowired
+    private UserDao userDao;
+
+    @Override
+    public void add(User user) {
+        userDao.add(user);
+    }
+
+    @Override
+    public List<User> listUsers() {
```

Comment:
You can rename listUsers

Reply:
This is not in the problem statement. I think it good name